thrift/atoms/tenfour.thrift

```diff
@@ -0,0 +1,61 @@
+namespace * tenfour
+
+include "../shared.thrift"
+
+/** date times are reprsented as i64 - epoch millis */
+typedef i64 DateTime
+
+struct Content {
```

Comment:
I agree that something like `TenfourContent` or `TenfourAtom` would be better than the generic `Content`, even if it's namespaced.